www/js/diary/services.js

```diff
@@ -509,15 +402,13 @@ angular.module('emission.main.diary.services', ['emission.plugin.logger',
                 startTs: result.end_ts - 10,
                 endTs: moment().unix() + 10
             }
-            var manualPromises = ConfirmHelper.INPUTS.map(function(inp) {
+            var manualPromises = manualInputFactory.MANUAL_KEYS.map(function(inp_key) {
               return UnifiedDataLoader.getUnifiedMessagesForInterval(
-                  ConfirmHelper.inputDetails[inp].key, pendingLabelQuery);
+                  inp_key, pendingLabelQuery).then(manualInputFactory.extractResult);
```

Comment:
Note to self: extractResult is primarily for enketo where we check for version number matching.